ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -13,7 +13,7 @@
  * @param helperTextDisabled="Allow tokens to be used indefinitely" {String} - This helper text is shown under the label when the toggle is switched off
  * @param helperTextEnabled="Disable the use of the token after" {String} - This helper text is shown under the label when the toggle is switched on
  * @param description="Longer description about this value, what it does, and why it is useful. Shows up in tooltip next to helpertext"
- * @param time=30 {Number} - The time (in the default units) which will be adjustable by the user of the form
+ * @param time='' {Number} - The time (in the default units) which will be adjustable by the user of the form
```

Comment:
I know, I actually already attempted to glimmerize at my look into the problem. Had the exact same thought you did, but thought just in case any backporting needs to be done, I'll keep these two separate.